api/server/router/network/network_routes.go

```diff
@@ -213,21 +209,11 @@ func (n *networkRouter) postNetworkCreate(ctx context.Context, w http.ResponseWr
 	}
 
 	if nws, err := n.cluster.GetNetworksByName(create.Name); err == nil && len(nws) > 0 {
-		return nameConflict(create.Name)
+		return libnetwork.NetworkNameError(create.Name)
 	}
 
 	nw, err := n.backend.CreateNetwork(create)
 	if err != nil {
-		var warning string
-		if _, ok := err.(libnetwork.NetworkNameError); ok {
-			// check if user defined CheckDuplicate, if set true, return err
-			// otherwise prepare a warning message
-			if create.CheckDuplicate {
```

Comment:
Do we still have other uses of this option? We should probably deprecated the field and the corresponding API parameter, and document that it's no longer used (not gated by API version).

Reply:
Oh, it's still in use in the daemon. This PR is only removing duplicated code.

Reply:
DOH! I should've read the actual PR description. I saw your post on slack about duplicated, and interpreted it as "duplicate error" (as in: duplicate name) 😂